for-jenkins/build-sbndaq_artdaq_core.sh

```diff
@@ -6,7 +6,7 @@ PROJECT_NAME=sbndaq-artdaq-core
 PROJECT_SOURCE_GIT_PREFIX=${PROJECT_SOURCE_GIT_PREFIX:-'https://github.com/sbnsoftware'}
 PRODUCTS=${PRODUCTS:-'/cvmfs/fermilab.opensciencegrid.org/products/artdaq:/cvmfs/fermilab.opensciencegrid.org/products/larsoft'}
 
-ARTDAQ_VERSION=${ARTDAQ_VERSION:-"v3_12_07"}
+ARTDAQ_VERSION=${ARTDAQ_VERSION:-"v3_12_06"}
```

Comment:
why the artdaq_version is downgrade here? thanks

Reply:
These come from originally pulling from develop and not v1_09_00. This and the other version skew are fixed.

Reply:
@dtorretta56  These are the two scripts (build-sbndaq_artdaq_core.sh, publishscisoft.sh) I use for running Jenkins builds, so it's safe to leave them as they are. I'll make adjustments myself if needed.

Reply:
Tanks Gennadiy . OK to proceed then.